keyboards/nuphy/air75_v2/ansi/info.json

```diff
@@ -0,0 +1,268 @@
+{
+    "keyboard_name": "NuPhy Air75 V2",
+    "manufacturer": "github.com/persama",
+    "bootloader": "stm32-dfu",
+    "processor": "STM32F072",
+    "usb":{
+        "vid": "0x19F5",
```

Comment:
This vendor ID is in use: 
https://www.the-sz.com/products/usbid/index.php?v=0x19F5&p=&n=

Reply:
To clarify, this vendor ID should be changed, as it belongs to Nationz Technology.

Please change the vendor ID. Either the generic 0xFEED (if not using via) or to something past 0x4000 (and the further past, the better).

(also, there are services, such as pid.codes that can assign a product id for you)

Reply:
Our company has chip products that use National Technology, and the ID number currently used is authorized by National Technology;

Reply:
@drashna  Regarding the issue of VID/PID, the one currently being used is authorized by Natios Tech; We also have other projects using this VID, which will not cause conflicts with other keyboards. Please confirm;